### ISSUE TITLE
[Search Result] Change the default URL generation

### ISSUE THREAD

**Issue by thiagarajanm** [OFFLINE-GmbH/oc-site-search-plugin#108]
How to change the default URL which is generated in the search result?
I have /category/blog-post. I have the blog category in the url as a dynamic field. 

The search result brings up /default or /post.

Could you help me with the change to get the category in the url?

**Comment by tobias-kuendig** [comment]
Blog post in what plugin? 

You can read here how to generate a page url if it is a custom plugin:
https://octobertricks.com/tricks/generate-link-cms-page

**Comment by thiagarajanm** [comment]
Its not a custom plugin. Its the rainlab.blog and rainlab.pages plugins.

**Comment by tobias-kuendig** [comment]
Did you set the correct `rainlab_blog_page` in SiteSearch's backend settings?

It should generate the link here:
https://github.com/OFFLINE-GmbH/oc-site-search-plugin/blob/33b47d278f23bebde8d99af402c16d5abe2eaa38/classes/providers/RainlabBlogResultsProvider.php#L76

by calling RainLab's setUrl method:
https://github.com/rainlab/blog-plugin/blob/master/models/Post.php#L157

That should help you debug the issue.

**Comment by thiagarajanm** [comment]
@tobias-kuendig  yes the backend settings are set correctly. But this only generated the default url set by october not the actual url used in the blog post page.

**Comment by tobias-kuendig** [comment]
I'm not sure what you mean. Can you post the page header of the cms page that hosts the blog component?

What URL is generated and what URL do you expect?

**Comment by tobias-kuendig** [comment]
I'm closing this issue due to inactivity. If you need any further assistance feel free to reply.
